healpy/src/_sphtools.pyx

```diff
@@ -83,7 +85,20 @@ def map2alm_spin_healpy(maps, spin, lmax = None, mmax = None):
     -------
     alms : list of 2 arrays
       list of 2 alms
+
+    Warnings
+    --------
+    This function previously made an exception for ``spin=0`` transforms, where
+    it flipped the sign of the output such that :math:`a_{lm}^+` matches the
+    output :math:`a_{lm}` of `map2alm`.  The new behaviour follows the HEALPix
+    convention, in which :math:`a_{lm}^+ = -a_{lm}` for a ``spin=0`` transform.
+
     """
+    spin = int(spin)
+
+    if spin == 0:
+        return [-map2alm(mm, niter=3, lmax=lmax, mmax=mmax) for mm in maps]
```

Comment:
I'm not sure about the `niter=3` call here; as far as I understand, iterative analysis is handled at a higher level and therefore I would specify `niter=0` here. If the toplevel `map2alm` was called with some non-zero `niter`, control flow will come past this location several times anyway.